packages/wonder-blocks-form/src/components/labeled-text-field.js

```diff
@@ -191,8 +176,10 @@ class LabeledTextFieldInternal extends React.Component<
             type,
             label,
             description,
+            value,
             disabled,
             validate,
+            onChange,
```

Comment:
Interesting 🤔 ... I have them in that order because that is how they are ordered in the [Implementation Spec](https://khanacademy.atlassian.net/wiki/spaces/FRONTEND/pages/1540423985/LabeledTextField+v1+Implementation+Spec). I would agree that ordering props in such "groups" would be cleaner. I'll keep this in mind for future cases, thanks!

Reply:
There's lots of different ways to order them. You could alphabetize them, group them based on whether or not they inherit from some base/common props, order them based on the ordering in other like-components (which is what it looks like in the spec... the ordering seems more clear there for some reason [to me]). I dunno... nothing you need to change, just something to think about.

Reply:
Yeah, thanks for the info!